packages/contentful--app-scripts/src/upload/build-upload-settings.ts

```diff
@@ -55,3 +56,7 @@ export async function buildAppUploadSettings(options: UploadOptions): Promise<Up
     ...appInfo,
   };
 }
+
+export function hostProtocolFilter (input: string) {
+  return input.replace(/^https?:\/\//, '');
```

Comment:
does this catch `http://` as well?

Reply:
It does indeed but I probably should have added a test case for http too huh

```
Welcome to Node.js v20.17.0.
Type ".help" for more information.
> "http://api.contentful.com".replace(/^https?:\/\//, '')
'api.contentful.com'
> "https://api.contentful.com".replace(/^https?:\/\//, '')
'api.contentful.com'
```

Reply:
added in 483466d575b08240e144359fc3794d80a4cffcf4